src/scripts/resources/airlock-lib.sh

```diff
@@ -65,13 +65,18 @@ function cleanUpHistory() {
     abortOnError
 
     echo "activate configuration..."
-    ${CURL} "https://${AIRLOCK}/airlock/rest/configuration/configurations/activate" \
+    STATUS_CODE=`${CURL} "https://${AIRLOCK}/airlock/rest/configuration/configurations/activate" \
```

Comment:
generally use of $() is prefered over back ticks

Reply:
DONE